theano/sandbox/cuda/basic_ops.py

```diff
@@ -3300,23 +3282,20 @@ def make_node(self, value, *shape):
                 const_shp = None
             bcast.append(numpy.all(1 == const_shp))
         otype = CudaNdarrayType(dtype='float32', broadcastable=bcast)
-        return Apply(self, [v] + sh, [otype()])
+        return Apply(self, sh, [otype()])
```

Comment:
I've thought about that solution as well, but I felt it would be a bit hacky. Anyway, the function would look like this:
```python
def return_true(*args):
    return True
```
But where would we put this? It seems misplaced in `theano.sandbox.cuda.basic_ops`.